src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/.github/workflows/build-release.yml

```diff
@@ -65,19 +67,137 @@ jobs:
           retention-days: "7"
           modern-installation: "true"
 
+  run-tests:
+    name: Run tests
+    runs-on: [windows-latest]
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+      fail-fast: false
```

Comment:
It might need to be longer than 10 minutes, but I think this should still include some value for `timeout-minutes`

Reply:
Added a timeout of 20 minutes.